blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json

```diff
@@ -1,16 +1,13 @@
 {
     "name": "Tether",
-    "website": "https://tether.to",
-    "description": "Tether (USDT) is a cryptocurrency with a value meant to mirror the value of the U.S. dollar.",
-    "explorer": "https://evm.confluxscan.net/address/0xfe97e85d13abd9c1c33384e796f10b73905637ce",
-    "type": "CFXEVM",
     "symbol": "USDT",
+    "type": "CFXEVM",
```

Comment:
```suggestion
    "type": "CONFLUX",
```

Reply:
@vcoolish in database we have CFXEVM for now. Should we rename it to CONFLUX ?

Reply:
@defisaur Yes, please